src/metabase/api/embed.clj

```diff
@@ -28,6 +28,7 @@
    [metabase.driver.common.parameters.operators :as params.ops]
    [metabase.models.card :as card :refer [Card]]
    [metabase.models.dashboard :refer [Dashboard]]
+   [metabase.models.params :refer [dashcards->param-to-field-ids-map]]
```

Comment:
Ah I originally didn't do this because params aliased namespace was already being used, and changing it to something that wasn't params gave a kondo warning. Should I just suppress the kondo error?

Reply:
Change isn't too big, will add in this pr